sporkfish/searcher/negamax.py

```diff
@@ -94,7 +99,8 @@ def _negamax(
                 board, depth, capture, move, alpha
             ):
                 board.pop()
-                self._statistics.increment_pruning()
+                # add test
```

Comment:
[Future] Can go?

Reply:
can't yet - haven't add test to check pruning